packages/core/src/browser/source-tree/tree-source.ts

```diff
@@ -37,12 +39,12 @@ export interface CompositeTreeElement extends TreeElement {
     getElements(): MaybePromise<IterableIterator<TreeElement>>
 }
 export namespace CompositeTreeElement {
-    /* eslint-disable @typescript-eslint/no-explicit-any */
-    export function is(element: CompositeTreeElement | any): element is CompositeTreeElement {
-        return !!element && 'getElements' in element;
+    export function is(element?: any): element is CompositeTreeElement {
```

Comment:
Since we do a `typeof` check, we can simplify it to `unknown`?

Reply:
Not really since TypeScript makes `object` type guards pretty painful to handle. `any` is fine here.